src/commands/Fun/leet.js

```diff
@@ -0,0 +1,17 @@
+exports.run = function(bot, message, args) {
+
+    if (params.length < 1) return message.channel.sendMessage(':x: Provide text to be leeted.')
```

Comment:
Please use brackets with if statements, and do not use `message.channel.sendMessage`. Instead just use `throw`:

```
throw 'Provide text to be leeted.'
```
It adds the :x: automatically.